packages/databases-collections/src/components/collection-fields/collection-fields.jsx

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable react/no-multi-comp */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import Checkbox from '@leafygreen-ui/checkbox';
+import TextInput from '@leafygreen-ui/text-input';
+import { Select, Option, Size } from '@leafygreen-ui/select';
+import IconButton from '@leafygreen-ui/icon-button';
+import Icon from '@leafygreen-ui/icon';
+
+import COLLATION_OPTIONS from '../../constants/collation';
+import hasTimeSeriesSupport from './has-time-series-support';
+import styles from './collection-fields.less';
+
+const HELP_URL_CAPPED = 'https://docs.mongodb.com/manual/core/capped-collections/';
+const HELP_URL_COLLATION = 'https://docs.mongodb.com/master/reference/collation/';
+
+const TIME_FIELD_INPUT_DESCRIPTION = 'Specify which field should be used ' +
+  'as timeField for the time-series collection. ' +
+  'This field must have a BSON type date.';
+
+const META_FIELD_INPUT_DESCRIPTION = 'The metaField is the designated field ' +
+  'for metadata.';
+
+const EXPIRE_AFTER_SECONDS_DESCRIPTION = 'The expireAfterSeconds field enables ' +
+  'automatic deletion of documents older than the specified number of seconds.';
+
+class FieldSet extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node
+  }
+
+  render() {
+    return (
+      <fieldset className={styles['form-group']}>
+        {this.props.children}
+      </fieldset>
+    );
+  }
+}
+
+class CollapsibleFieldset extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    label: PropTypes.string.isRequired,
+    description: PropTypes.oneOfType([
+      PropTypes.element,
+      PropTypes.string
+    ]),
+    helpUrl: PropTypes.string,
+    onToggle: PropTypes.func.isRequired,
+    toggled: PropTypes.bool,
+    openLink: PropTypes.func
+  }
+
+  render() {
+    return (
+      <FieldSet>
+        <Checkbox
```

Comment:
When I run it locally, this checkbox looks pretty weird:

<img width="188" alt="image" src="https://user-images.githubusercontent.com/5036933/121502920-e2365180-c9e0-11eb-8d34-4bbffd22bab1.png">

Reply:
We figured it out! It's a zooming issue